configs/lima_bananapro_defconfig

```diff
@@ -0,0 +1,57 @@
+BR2_arm=y
+BR2_cortex_a7=y
+BR2_ARM_FPU_VFPV4=y
+BR2_JLEVEL=4
```

Comment:
BR2_JLEVEL should be kept as 0 in the defconfig.

Reply:
Hy Erico.

1) BR2_JLEVEL should go with system defaults. I just wasn't aware of this Buildroot default. Thanks for pointing out.

2) linux-lima.config is handling the A20 based BananaPro. I just added a seperate config to devide it for the given default  (e.g an usb NIC, external disk with btrfs, etc.). Your default is enough to get the internel devices up (nic, sound, usb, sata). 

cheers
ralf

Reply:
Sounds good! Can you update the pull request with the BR2_JLEVEL and the BananaPro using the same linux-lima.config so this can be merged?
If you want you can add some more modules to the linux-lima.config too, but I would prefer not having two linux configs unless absolutely necessary.